src/mixins/commands_mixin.js

```diff
@@ -219,7 +218,29 @@ export const commands_mixin = {
         {
           ...num_focus_pts && { num_focus_pts }
         },
-        {}
+        {
+          filter: this.filter_wheel_options_selection,
+          bin: this.camera_bin
+        }
+      )
+    },
+    focus_adjust_command () {
+      return this.base_command(
+        'focuser',
+        'focusadjust',
+        '',
+        {},
+        {
+          filter: this.filter_wheel_options_selection,
+          bin: this.camera_bin
+        }
+      )
+    },
+    zcompress_command () {
```

Comment:
What exactly is the z compress button/command for? I'm not sure I understand the purpose, if it's for compressing images, sites already do this step. Is it for something else that I missed?